portfolio/src/main/webapp/style.css

```diff
@@ -44,6 +44,16 @@ body {
     font-family: 'Roboto Mono', monospace; 
 }
 
+/* style h2 */
+h2 {
+    font-weight: normal;
+}
+
+/* style h4 */
+h4 {
+    font-weight: normal;
+}
+
```

Comment:
Same as the other PR, if you fix it there and merge it, I expect pulling will fix it here too.